node/service/src/lib.rs

```diff
@@ -100,7 +100,7 @@ pub use sp_api::{ApiRef, ConstructRuntimeApi, Core as CoreApi, ProvideRuntimeApi
 pub use sp_runtime::{
 	generic,
 	traits::{
-		self as runtime_traits, BlakeTwo256, Block as BlockT, HashFor, Header as HeaderT, NumberFor,
+		self as runtime_traits, BlakeTwo256, Block as BlockT, HashingFor, Header as HeaderT, NumberFor,
```

Comment:
```suggestion
		self as runtime_traits, BlakeTwo256, Block as BlockT, Header as HeaderT, NumberFor,
```

Not sure we need this here, if this is not used anywhere else.